backend/app/services/attachment/parser.py

```
@@ -972,6 +981,54 @@ def _parse_image(self, binary_data: bytes, extension: str) -> Tuple[str, str]:
+        Args:
+            binary_data: Archive file binary data
+            extension: File extension (e.g., '.zip')
+
+        Returns:
+            Metadata text describing the archive
+        """
+        try:
+            archive_file = io.BytesIO(binary_data)
+
+            # Build description
+            text = f"[压缩文件]\n"
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)